walkthroughs/week-4-tdd/project/src/test/java/com/google/sps/FindMeetingQueryTest.java

```diff
@@ -57,7 +57,7 @@ public void setUp() {
   }
 
   @Test
-  public void optionsForNoAttendees() {
+  public void optionsForNoAttendees() { //done
```

Comment:
what do you think about removing these //done comments?

Reply:
Yes! I will! I used it as a pointer to help me know what I completed and what I didn't but now I can take them off